pkg/controller/vsphere/reconciler.go

```diff
@@ -184,6 +188,10 @@ func (r *Reconciler) reconcileMachineWithCloudState(vm *virtualMachine) error {
 	klog.V(3).Infof("%v: reconciling machine with cloud state", r.machine.GetName())
 	// TODO: reconcile task
 
+	if err := r.reconcileRegionAndZoneLabels(vm); err != nil {
+		klog.Errorf("Failed to reconcile region and zone labels: %v", err)
```

Comment:
don't we want to error here?
we might want to adapt `reconcileMachineWithCloudState` to return NewAggregate(errlist []error)

Reply:
We can, but this seems totally optional. I thought I would rather have a working machine without region and zone labels (that are only used by the column printers in the CRD) than to fail here, but I honestly don't have a strong opinion.